### PR TITLE
feat:button done

### PR DESCRIPTION
**Pull request by Developers-codz**

### CODE REVIEW

File: css/component/button.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    cursor:pointer;
+}
+.btn-outline{
+    border:1px solid var(--primary-color);
+    background: transparent;
+    color:var(--primary-color)
+}
+.rm-border{
+    border:none;
+}
```
Comment (Dev-AkshatGupta) [inline css/component/button.css:14]: Use better naming This rm looks bit awkward can be rem instead It seems to be much better understandability

File: css/component/utility.css [hunk 2]
```diff
@@ -28,21 +28,7 @@
 .vertical-direction{
     flex-direction: column;
 }
-.btn{
-    cursor: pointer;
-}
```
Comment (Dev-AkshatGupta) [inline css/component/utility.css:33]: You can instead reset for this instead of making a class

File: css/component/button.css (hunk 1 above)
Comment (Developers-codz) [reply css/component/button.css:14]: thanks for reviewing Akshat but I think using rem can create confusion as its unit for font size

File: css/component/utility.css (hunk 2 above)
Comment (Developers-codz) [reply css/component/utility.css:33]: ok! I will do that.

**Comment by anik31** [comment]
Looks good

**Pull request merged**